File: IT/2/Input-Output/Esercizi/Python/esercizio8.py
# file esercizio8.py

from __future__ import print_function
if hasattr(__builtins__, 'raw_input'): input = raw_input
from datetime import datetime
from os import linesep

filename = input("Apri il file: ")
f = open(filename, "w")

date = datetime.now()

f.write("File creato il " + date.strftime("%d-%m-%Y alle ore %H:%M:%S") + linesep)
f.close()

File: IT/2/Input-Output/Esercizi/C++/esercizio8.cpp
/*
file esercizio8.cpp
compilare con: g++ -std=c++11 esercizio8.cpp
*/

#include <ctime>
#include <iostream>
#include <fstream>

using std::cin;
using std::cout;
using std::endl;
using std::string;
using std::ofstream;

int main()
{
	cout << "Apri il file: ";

	string filename;
	getline(cin, filename);

	ofstream f(filename);

	time_t now = time(NULL);
	struct tm *timeinfo = localtime(&now);
	char buffer[80];
	strftime(buffer, sizeof(buffer), "File creato il %d-%m-%Y alle ore %H:%M:%S", timeinfo)\
;

	f << string(buffer) << endl;
}

